package.json

```diff
@@ -64,41 +64,38 @@
     "url": "https://github.com/aeternity/aepp-sdk-js"
   },
   "devDependencies": {
-    "@babel/cli": "^7.16.8",
-    "@babel/core": "^7.16.12",
+    "@babel/cli": "^7.17.0",
+    "@babel/core": "^7.17.2",
     "@babel/node": "^7.16.8",
-    "@babel/plugin-transform-runtime": "^7.16.10",
+    "@babel/plugin-transform-runtime": "^7.17.0",
     "@babel/preset-env": "^7.16.11",
     "@babel/preset-typescript": "^7.16.7",
-    "@babel/register": "^7.16.9",
+    "@babel/register": "^7.17.0",
     "@commitlint/cli": "^16.1.0",
     "@commitlint/config-conventional": "^16.0.0",
     "@types/bn.js": "^5.1.0",
     "@types/chai": "^4.3.0",
     "@types/mocha": "^9.1.0",
-    "@types/node": "^17.0.10",
-    "@typescript-eslint/eslint-plugin": "^5.10.1",
-    "@typescript-eslint/parser": "^5.10.1",
+    "@types/node": "^17.0.16",
+    "@typescript-eslint/eslint-plugin": "^5.11.0",
+    "@typescript-eslint/parser": "^5.11.0",
     "babel-loader": "^8.2.3",
     "babel-plugin-add-import-extension": "^1.6.0",
     "babel-plugin-transform-default-named-imports": "^1.2.2",
-    "chai": "^4.3.4",
+    "chai": "^4.3.6",
     "chai-as-promised": "^7.1.1",
     "codecov": "^3.8.3",
     "eslint": "^7.32.0",
```

Comment:
what about eslint 8?

Reply:
https://github.com/standard/eslint-config-standard/issues/209
didn't know that they have [17.0.0-1](https://www.npmjs.com/package/eslint-config-standard/v/17.0.0-1) (on next tag), switching to it